test/schemas/displayModes/Category.Mode.display

```diff
@@ -0,0 +1,4 @@
+DisplayMode({
```

Comment:
`Category.display` not `Category.Mode.display`

Reply:
`Category.sidplay` is not valid, because it is located in root directory of schemas, thus it should have the form: `categoryName.displayModeName.display`.